phaser/util.py

```diff
@@ -1,22 +1,11 @@
-import pandas as pd
+from clevercsv import stream_dicts, stream_table
+from .exceptions import DataErrorException
 
 
 def read_csv(source):
-    """ Includes the default settings phaser uses with panda's read_csv. Can be overridden to provide
-    different read_csv settings.
-    Defaults:
-    * assume all column values are strings, so leading zeros or trailing zeros don't get destroyed.
-    * assume ',' value-delimiter
-    * skip_blank_lines=True: allows blank AND '#'-led rows to be skipped and still find header row
-    * doesn't use indexing
-    * does attempt to decompress common compression formats if file uses them
-    * assume UTF-8 encoding
-    * uses '#' as the leading character to assume a row is comment
-    * Raises errors loading 'bad lines', rather than skip
-    """
-    return pd.read_csv(source,
-                       dtype='str',
-                       sep=',',
-                       skip_blank_lines=True,
-                       index_col=False,
-                       comment='#')
+    stream = stream_table(source)
+    header = next(stream)
```

Comment:
Without reading clevercsv's documentation or code, I'm assuming that `stream_table(source)` followed by `next(stream)` only reads the first line and does not read the entire file.  Is that correct?  (A reasonable answer is "RTFM.")

Reply:
Yes you're right and I'll leave a comment here to help future readers not have to RTFM or guess.